homework_9/main.py

```diff
@@ -0,0 +1,66 @@
+"""
+Напишите следующие функции:
+* Нахождение корней квадратного уравнения
+* Генерация csv файла с тремя случайными числами в каждой строке. 100-1000 строк.
+* Декоратор, запускающий функцию нахождения корней квадратного уравнения с каждой тройкой чисел из csv файла.
+* Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл.
+"""
+import csv
+import json
+from pathlib import Path
+from random import randint as rnd
+
+CSV_PATH = Path.cwd() / 'numbers.csv'
+JSON_PATH = Path.cwd() / 'result.json'
+
+
+def deco_quadratic(func):
+    def wrapper():

```

Comment:
Помните, я говорил на семинаре, что функция обертка должна как бы "мимикрировать" под оригинальную функцию, то есть она должна принимать и возвращать то же самое, что и декорируемая функция? но у вас обертка ничего не принимает
```suggestion
    def wrapper(*args):

```